modules/installation-rhv-specifying-assets-directory.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-specifying-assets-directory_{context}"]
+= Specifying the assets directory
```

Comment:
Why can't this be part of Setting up the installation machine?

Reply:
I agree.